WPUserManager/wp-user-manager
# Allowed Roles in the Custom Field Permissions tab needs to have at least one role

Resolves #378 

## Description

## Testing Instructions

1. Go to Custom Fields
2. Edit a custom field
3. Go to Permissions tab
4. Add a role in the Allowed Roles field, then save it
5. From here, edit the field again, and try to delete the role, then save
6. Check the field again, and see that the role is not deleted.

## Pre-review Checklist

- [ ] [Issue and pull request titles](https://github.com/WPUserManager/development-handbook/blob/master/issue-pr-titles.md) are properly formatted.
- [ ] [Acceptance criteria](https://github.com/WPUserManager/development-handbook/blob/master/acceptance-criteria.md) have been satisfied and marked in the related issue.
